### DISCUSSION TITLE
1.1.1 업데이트 후 워닝 문의

### DISCUSSION THREAD

**Discussion by mhkim0720** [naverz/zepeto-studio-kor#121] [5. Q&A]
안녕하세요 

어제 월드 1.1.1 업데이트후 런타임에 아래 이미지와 같은 워닝이 에디터에 계속 뜨는데 원인이 뭘까요? 

![image](https://user-images.githubusercontent.com/8485731/147176257-a7b7d6ae-d98b-4a78-b82c-7bd4fc51d6a6.png)

한 두개 뜨면 무시하겠는데 런타임에 저 워닝이 계속 쏟아 집니다. ; 

메세지 보면 캐릭터 애니메이터에 포함되지 않은 파라미터를 호출했다는 뜻 같은데 

업데이트 전에는 없던 워닝 이라 문의 드립니다. 

애니메이터는 기본 제공 되는 애니메이터 외에  앉기/수영하기 애니메이터를 RuntimeAnimatorController 를 이용해 

스위칭 하며 사용 중이기는 한데 중요한 건 저 두개 애니메이터를 스위칭 하기전 기본 제공 애니메이터

상에서 캐릭터를 움직이기만 해도 저 워닝이 쏟아 집니다 .  

무시해도 되는 워닝인지 해결 방법이 있는지 

답변 부탁드립니다.

**Comment by cheolhui-lee** [comment]
Character Controller 쪽 업데이트가 있었는데 연관이 있었는지 확인해보겠습니다.

**Comment by moondory77-naverz** [comment]
안녕하세요. 
이번 world 1.1.1 업데이트에서, 캐릭터의 이동 모션을 향상시키기위한 기능이 추가되었습니다.
 
이 때, ZepetoCharacter 컴포넌트가 update에 제어하는 animator asset 파라메터가 일부 추가되었는데요.

1.1.1 업데이트 후, (이 부분이 반영되지 않은) 기존 animator asset을 사용할 때 이러한 warning이 발생할 수 있습니다.

동작 자체에는 문제 없지만, 
ZepetoPlayers 컴포넌트의 Player >> AnimatorController에 지정된 animator asset을
업데이트 된 CharacterController 패키지 내, animator asset으로 교체한다음 사용부탁드리겠습니다.
(Packages > ZEPETO.CharacterController > Runtime > _Resources > AnimatorController > ZepetoAnimator.controller)

다음 업데이트에, 이러한 animator asset의 호환성을 보장할 수 있도록 추가 업데이트를 진행하겠습니다.
패키지 버전간 호환성에 조금 더 유념하며 업데이트를 진행하겠습니다.

감사합니다.
